### ISSUE TITLE
PHIL and SOCI courses aren't shown as counting for H/SS

### ISSUE THREAD

**Issue by arunim1** [upenn/irs#7]
at least phil 1000, 1330, soci 001 were not counted, when https://ugrad.seas.upenn.edu/student-handbook/courses-requirements/social-sciences-and-humanities-breadth/ says they do in fact count.

**Comment by devietti** [comment]
Right, this is essentially the same as #2 and due to the ambiguity of the Handbook's phrasing that SOCI courses are all Social Science except:
> statistics, probability, and math courses

And similarly, all PHIL courses are Humanities except:
> 005, 006, and all other logic courses

where the exact list of non-SS/non-H courses is left unspecified.
